examples/darwin-framework-tool/main.mm

```diff
@@ -26,6 +27,7 @@
 #include "commands/delay/Commands.h"
 #include "commands/discover/Commands.h"
 #include "commands/interactive/Commands.h"
+#include "commands/leaks/Commands.h"
```

Comment:
This file does not seem to exist... should the changes here not be in this PR?

Reply:
Oops sorry. I missed this one when rebasing.